avida-ts/package.json

```diff
@@ -0,0 +1,59 @@
+{
+  "name": "avida-ts",
+  "private": true,
+  "version": "1.0.0",
+  "description": "",
+  "scripts": {
+    "ci:lint": "pnpm lint",
+    "ci:check": "pnpm --recursive run check",
+    "setup": "pnpm run reset && pnpm install",
+    "reset": "pnpm dlx rimraf --glob ./**/node_modules",
+    "build": "NODE_ENV=production pnpm run --recursive --stream build",
```

Comment:
Usually, if there are some jobs like `build:txutils` or others it's assumed that `build` action combines all the things together, like:
```
"build": "pnpm run build:txutils && pnpm run build:types && ..."
```
I think it worth to change `build` job into something like `build:production` or other thing which points to exact action.

Reply:
fixedin https://github.com/nymlab/AVIDA/pull/6/commits/7ce3c1886296d88d19edcfd650036b81c8dcdf75